Makefile

```diff
@@ -77,7 +78,22 @@ clean:
 	rm -rf translate/dist .server-build
 
 shell:
-	"${DC}" run --rm server //bin/bash
+	@container=$$(${DOCKER} ps -q --filter ancestor=local/pontoon | head -n 1); \
+	if [ -z "$$container" ]; then \
+  		echo "Error: No container found based on local/pontoon" >&2; \
+  		exit 1; \
+	else \
+  		${DOCKER} exec -it $$container /bin/bash; \
```

Comment:
When I exit the shell with Ctrl+D, I get a promotional message:

```
What's next:
    Try Docker Debug for seamless, persistent debugging tools in any container or image → docker debug ccebe29e4262
    Learn more at https://docs.docker.com/go/debug-cli/
```

Could you please see if [`DOCKER_CLI_HINTS`](https://forums.docker.com/t/how-to-turn-off-the-whats-new-message/140860/6) indeed removes it?